src/serverMain/kotlin/com/studo/campusqr/endpoints/AccessManagement.kt

```diff
@@ -149,6 +149,11 @@ suspend fun AuthenticatedApplicationCall.createAccess() {
 
   runOnDb {
     getCollection<BackendAccess>().insertOne(newAccess, upsert = false)
+
+    // Set location to restricted, otherwise the access control doesn't make sense
```

Comment:
Is this the right place to do that? Shouldn't we handle this case in the UI and tell the user that he can't create an AccessControl on that location since the location is not restricted? Doing on an access-creation some "side-effects" could lead to bad UX.